bin/export_atlas_uniprot.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# What this script does:
+# 1. Gets all unique UniProt IDs in Atlas bulk-analytics collection
+# 2. Generates an Atlas search linnk for each UniProt ID
+# 3. Writes everything to file
+
+# Caveats:
+# 1. Only works for bulk-analytics for now. There seems to be no  in scxa solr (???) still waiting for reply from devs
+# 2. Only gets 100 unique uniprot IDs...seems odd.  Still investigating this.
+
+
+atlasUniqueUniprotUrl="http://${SOLR_HOST}/solr/bulk-analytics-v1/select?facet.field=keyword_uniprot&facet=on&q=*:*&rows=0&start=0"
+ATLAS_UNIPROT_EXPORT_FILE=${$1:-$ATLAS_UNIPROT_EXPORT_FILE}
```

Comment:
What if `$1` is not defined and `$ATLAS_UNIPROT_EXPORT_FILE` not defined as well..

Generally here a default hardcoded filename given

Reply:
Done in 14a35a1dbc67eb413208241473b93f3456bfe6ed.